test/common/config_test.js

```diff
@@ -91,7 +99,7 @@ test('config parseConfigJson should return an empty json in ENV_MOCK', (t) => {
     t.deepEqual(configJson, {});
 });
 
-test('config parseConfigJson should return an empty json for a missing temp file in ENV_DEV', (t) => {
+test('config parseConfigJson should return an empty json for aanswer for the Cromwell dev environment missing temp file in ENV_DEV', (t) => {
```

Comment:
I don't understand what this is trying to say. Maybe this is an unintentional change? At a minimum, there's an extra "a".